### ISSUE TITLE
Can anyone help me write README in detail?

### ISSUE THREAD

**Issue by xalanq** [xalanq/cf-tool#63]
I need a very detailed README. Can anyone help me ~

**Comment by Nojus297** [comment]
I could try to, but so much has change over the few last days, that I'm not even sure in which direction this project is moving (like the new contests folder instead of working dir).
If we could first establish solid and persistent API and usage, we could then try to write an informative and comprehensive README.

**Comment by infixint943** [comment]
I could help you for sure. But the latest update has made quite a few changes that seem to make the tool a lot more complicated. Once I fully understand the updates to the tool, maybe I could help you write the README

**Comment by xalanq** [comment]
Thank you 2 guys!

**Comment by etanot** [comment]
Why README? README is suppose to be [short](https://opensource.guide/starting-a-project/#writing-a-readme) introduction to a project. Isn't it a good idea to use **[Github Wiki](https://help.github.com/en/github/building-a-strong-community/documenting-your-project-with-wikis)** and for offline documentation in UNIX and UNIX-like OS, **[man page](https://en.wikipedia.org/wiki/Man_page)** instead of putting everything in README?

**Comment by gnojus** [comment]
I agree that README shouldn't be a super long document, but I don't thing that project of this scale needs a wiki. Maybe README should explain the behavior and ideology, and commands should accept --help flag? E. g. `cf clone --help` gives usage explanation of command `clone`.

**Comment by etanot** [comment]
> I don't thing that project of this scale needs a wiki

We already crossed that scale, we have around 3-4 pager longer usage guideline, FAQ, Template etc everything is in README, which ultimately makes it cluttered. Putting them in Wiki, would make things structured and unlike README, Wiki requires very less effort to edit, that is whenever we make a small changes like typo fix, it requires to clone, commit, push, merge etc; and all those steps which require to make MR for **real code** change and unnecessarily those will present in code repository's history; but with Wiki we have our own separate history, and we can easily edit anything using Github UI, just by using few clicks.

> commands should accept --help flag

also `help` as sub-command like git, go etc does. For example `cf help clone`.

**Comment by infalmo** [comment]
@nojus297 Would you be interested in helping me port this tool to other websites too? I've started the work in my organisation `cp-tools` and would be glad if you could help!

**Comment by lifeModder19135** [comment]
Is this project dead?  There is a MONSTER of a bug in the (config) directory parsing logic (so much so that I didn't even get to register for today's competition, as I decided to try out `cf config` / `cf race ...` and spent the 2 hrs trying  -- unsuccessfully so far -- to fix my computer instead), but I don't know if I should waste the energy reporting. There are currently 51 issues, and only 2 have been closed within the last year. There is no point in building something like this if nobody is going to maintain it. It is the same as if a group of boat builders were to spend their days building a massive cargo liner just to shove it off and watch it sink into the middle of the ocean. I hate seeing open-source projects with potential gathering dust. It's a damned shame.  

If it needs a maintainer, I can offer a bit of my time, although I have projects of my own. Still, it hurts to see a project like this rotting...
